QC/wdl/workflows/SV_assembly.wdl

```diff
@@ -8,6 +9,15 @@ import "../tasks/jasmine.wdl" as runJasmine
 workflow sv_assembly{
     
     input{
+        File ParlInputBam
+        File ParlRefGenome
```

Comment:
I am not sure, but I think that these inputs need to be optional at the workflow level (since running parliament is optional).